### PR TITLE
Remove ngx-translate and switch to framework agnostic inputs

### LINKED ISSUE #63: Why i18n and @ngx-translate in this library ?
I'm wondering why UI component library at this level handling its own i18n texts and moreover dictating which i18n library to use ? Aren't i18n texts and which i18n library to use are completely for the top level application to provide and decide instead, and this library itself should stay agnostic ? 

References and comparisons : 
- https://github.com/angular-ui/bootstrap
- https://github.com/valor-software/ngx-bootstrap
 
Thank You

### PR DESCRIPTION
**Pull request by cal-smith**
closes #63

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -65,22 +63,21 @@
 			"RESET_SEARCH": "Reset search"
 		}
 	},
-	"MODAL": {
+	"MODAL" : {
```
Comment (zvonimirfras) [inline src/i18n/en.json]:
```suggestion
	"MODAL": {
```

File: src/i18n/en.json [hunk 2]
```diff
@@ -65,22 +63,21 @@
...
 		"CLOSE": "Close modal"
 	},
-	"OVERFLOW_MENU": {
+	"OVERFLOW_MENU" : {
```
Comment (zvonimirfras) [inline src/i18n/en.json]:
```suggestion
	"OVERFLOW_MENU": {
```

File: src/pagination/pagination.component.ts [hunk 3]
```diff
@@ -149,6 +147,8 @@ export class Pagination {
 	 */
 	@Input() model: PaginationModel;
 
+	@Input() translations = EN.PAGINATION;
```
Comment (zvonimirfras) [inline src/pagination/pagination.component.ts]: Missing service?
```suggestion
	@Input() translations = this.i18n.get().PAGINATION;
```

File: src/dialog/overflow-menu/overflow-menu.component.ts [hunk 4]
```diff
@@ -43,7 +44,10 @@ import { Component, ElementRef } from "@angular/core";
 	`]
 })
 export class OverflowMenu {
-	constructor(private elementRef: ElementRef) {}
+
```
[the remark this answers is not in the document]
Comment (cal-smith) [reply src/dialog/overflow-menu/overflow-menu.component.ts]: I think it's just your diff ... I don't see that locally

File: src/pagination/pagination.component.ts (hunk 3 above)
Comment (cal-smith) [reply src/pagination/pagination.component.ts]: Oops, good catch

**Comment by cal-smith** [comment]
I'm not sure if we want to tackle the calendar stuff now given that we're going to be refactoring it anyway.

I was thinking about some of the potential weirdness with the i18n service ... I think the best fix would be to return an Observable that updates with the correct value ... thinking about it I've got a couple ideas that could work pretty well with that ... and thanks to the `async` pipe it's pretty easy to handle in templates

File: src/banner/banner.component.ts [hunk 5]
```diff
@@ -38,7 +40,7 @@ import { NotificationContent } from "./banner-content.interface";
 			type="button">
 			<svg
 				class="bx--inline-notification__close-icon"
-				aria-label="close"
+				[attr.aria-label]="bannerObj.closeLabel"
```
Comment (youda97) [inline src/banner/banner.component.ts]: this label should be on the button to fix DAP violation: `Form controls must have associated labels`. See #194 for more clarification

File: src/banner/toast.component.ts [hunk 6]
```diff
@@ -28,7 +28,7 @@ import { Banner } from "./banner.component";
 			(click)="onClose()">
 			<svg
 				class="bx--toast-notification-icon"
-				aria-label="close"
+				[attr.aria-label]="bannerObj.closeLabel"
```
Comment (youda97) [inline src/banner/toast.component.ts]: this label should be on the button to fix DAP violation: `Form controls must have associated labels`. See #194 for more clarification

File: src/banner/banner.component.ts (hunk 5 above)
Comment (cal-smith) [reply src/banner/banner.component.ts]: if it's in that PR I don't need to fix it here as well.

**Pull request merged**